is_core/auth_token/middleware.py

```diff
@@ -26,23 +27,42 @@ def process_request(self, request):
         request.user = SimpleLazyObject(lambda: get_user(request))
         request._dont_enforce_csrf_checks = utils.dont_enforce_csrf_checks(request)
 
+    def _update_token_and_cookie(self, request, response, max_age, expires):
+        request.token.save()
+        response.set_cookie(settings.AUTH_COOKIE_NAME, force_text(request.token.key), max_age=max_age,
+                            expires=expires, httponly=settings.AUTH_COOKIE_HTTPONLY,
+                            secure=settings.AUTH_COOKIE_SECURE, domain=settings.AUTH_COOKIE_DOMAIN)
+        return response
+
+    def _set_auth_expiration_header(self, request, response):
+        response[settings.AUTH_TOKEN_EXPIRATION_HEADER] = request.token.time_to_expiration
```

Comment:
už dokáži lépe vysvětlit proč se mi ten python formát http názvu hlaviček v configu nelíbí

když se bere hlavička z request musí být django formát (upper s podtržítkama místo pomlček). Když se ale header nastavuje do response tak to tak být nemusí